components/script/dom/bindings/htmlconstructor.rs

```diff
@@ -222,7 +222,7 @@ unsafe fn html_constructor(
 }
 
 /// Returns the constructor object for the element associated with the given local name.
-/// This list should only include elements marked with the [HTMLConstructor] extended attribute.
+/// This list should only include elements marked with the \[HTMLConstructor\] extended attribute.
```

Comment:
I escape the brackets isn't quite the right fix here. We either want to link to the Rust data structure, or if it's not a Rust data structure we probably want to use backticks intead.

Reply:
Attempted using backticks, but encountered the same warning. Setting an explicit link to [HTMLConstructor](https://html.spec.whatwg.org/multipage/dom.html#htmlconstructor) indeed resolved the issue